Octokit.Tests/Clients/EventsClientTests.cs

```diff
@@ -223,6 +400,7 @@ public void RequestsCorrectUrl()
                 await Assert.ThrowsAsync<ArgumentException>(() => client.GetAllForAnOrganization("", "org"));
                 await Assert.ThrowsAsync<ArgumentNullException>(() => client.GetAllForAnOrganization("fake", null));
                 await Assert.ThrowsAsync<ArgumentException>(() => client.GetAllForAnOrganization("fake", ""));
+                await Assert.ThrowsAsync<ArgumentNullException>(() => client.GetAllForAnOrganization("fake", "org", null));
```

Comment:
same thing as above - check `("", "org", ApiOptions.None)` and ("fake", "", ApiOptions.None)`